waspc/cli/Wasp/Cli/Command/Db/Migrate.hs

```diff
@@ -1,31 +1,25 @@
 module Wasp.Cli.Command.Db.Migrate
   ( migrateDev,
-    copyDbMigrationsDir,
-    MigrationDirCopyDirection (..),
   )
 where
 
-import Control.Monad.Catch (catch)
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 -- Wasp generator interface.
 
-import qualified Path as P
-import qualified Path.IO as PathIO
-import StrongPath (Abs, Dir, Path', (</>))
-import qualified StrongPath.Path as SP.Path
+import StrongPath ((</>))
 import Wasp.Cli.Command (Command, CommandError (..))
 import Wasp.Cli.Command.Common
   ( findWaspProjectRootDirFromCwd,
     waspSaysC,
   )
 import qualified Wasp.Cli.Common as Cli.Common
 import Wasp.Cli.Terminal (asWaspFailureMessage, asWaspStartMessage, asWaspSuccessMessage)
-import Wasp.Common (WaspProjectDir, dbMigrationsDirInWaspProjectDir)
-import Wasp.Generator.Common (ProjectRootDir)
-import Wasp.Generator.DbGenerator (dbMigrationsDirInDbRootDir, dbRootDirInProjectRootDir)
 import qualified Wasp.Generator.DbGenerator.Operations as DbOps
 
+-- | Note(shayne): Migrates using migrations dir in project dir. The source down to project
+-- migrations dir copy process happens automatically during web app generation.
+-- The copy back up happens in DbGenerator.Operations.migrateDevAndSync.
```

Comment:
If I got it correctly the actual copying is happening here (in this file, line 32?), but what you meant is that the logic for that is located in `DbGenerator.Operations.migrateDevAndSync`, right?

Maybe we don't need to specify an exact source location of the function used within this function (that can get out-of-date), but rather say sth like  "This function takes care of copying migrations from the generated project back to the source code", so we make it more like behavior explanation (and less likely to get outdated).

On the other hand, from `migrateDevAndSync` name is not immediately clear what syncing refers to, is that why you added this explanation? Maybe a more appropriate name would be `migrateDevAndSyncWithSource`? Or even more bluntly, `migrateDevAndCopyToSource` since there isn't much syncing (sounds like a 2-way process) going on, we just overwrite everything in source, right?

I put lof of stuff now here :D. No issue here really, just providing some feedback from the "outer" perspective in case it is helpful.

Reply:
Yeah good call on not explicitly naming the function. I can update the comment and consider different names! 👍🏻

Reply:
Updated comment as follows:
```haskell
-- | NOTE(shayne): Performs database schema migration (based on current schema) in the generated project.
-- This assumes the wasp project migrations dir was copied by a previous compile.
-- The migrate function takes care of copying migrations from the generated project back to the source code.
```

and changed function name to `migrateDevAndCopyToSource` in this commit https://github.com/wasp-lang/wasp/pull/420/commits/42a03534386d45f19aed9889088fb3f05ccf9f85